examples/research_projects/wav2vec2/run_training.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
```

Comment:
@LysandreJik, @sgugger 

I think it makes sense here to first put everything under `research_projects` and once it's heavily used we can maybe move the collator to `src/transformers` and move this script under `examples/speech`

Reply:
Yes, this works for me.

Reply:
I 100%-agree